src/products/products.jsx

```diff
@@ -0,0 +1,144 @@
+import React, { useState } from 'react';
+import './products.css'
+import { Link } from 'react-router-dom';
+
+const Allproducts = [
```

Comment:
why did you decide to name this variable with capital letter? capital letters are usually used to name classes e.g. `class Product`
```suggestion
const allProducts = [
```